x-pack/plugins/security_solution/public/timelines/components/open_timeline/note_previews/index.tsx

```diff
@@ -227,7 +226,7 @@ export const NotePreviews = React.memo<NotePreviewsProps>(
                     size="l"
                   />
                 ),
-                actions: <EditTimelineButton timelineId={timelineId} initialFocus="description" />,
```

Comment:
This will only remove the edit button from the "first" note which is the description. It was a request from product and design to remove this button as well. It's mentioned in https://github.com/elastic/kibana/issues/165053 but further down :)

Reply:
Thanks for explanation @janmonschke